src/_components/AssetTypes/AddAssetTypesContainer.jsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import AddAssetTypesComponent from '../../components/AssetTypes/AddAssetTypesComponent';
+import { ToastMessage } from '../../_utils/ToastMessage';
+
+import { loadSubCategories } from '../../_actions/subcategory.actions';
+import { createAssetType } from '../../_actions/assetTypes.actions';
+import resetToastMessageContent from '../../_actions/toastMessage.actions';
+
+export class AddAssetTypesContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
The constructor use is more ideal if binding methods or initializing state using a prop value. Therefore, declare the state as a class variable. I.e.

*Necessary use of constructor*
```
export class AddAssetTypesContainer extends React.Component {
  constructor(props) {
    this.state = {
      myState: props.myProp
    }
    this.handleFunction = this.handleFunction.bind(this);
  }
  ...
  ...
}
```
*Note*: Use of [props to initialize state is usually discouraged](https://reactjs.org/docs/react-component.html#constructor), unless you want to ignore the prop update

*State declared as a variable*
```
export class AddAssetTypesContainer extends React.Component {
  state = {
     myState: 0
  };
  ...
  ...
  render() {...}
}
```

Reply:
@sachungo oops I had forgotten to remove the unnecessary `updateToastMessageContent` in `assetTypes.actions.js`.

I see the point when to use and not use the constructor. Adjusting the code.